src/core/syscalls/business_logic_syscall_handler.rs

```diff
@@ -16,16 +17,23 @@ use num_bigint::BigInt;
 
 pub struct BusinessLogicSyscallHandler {
     tx_execution_context: Rc<RefCell<TransactionExecutionContext>>,
+    general_config: RefCell<StarknetGeneralConfig>,
     events: Rc<RefCell<Vec<OrderedEvent>>>,
+    tx_info_ptr: RefCell<Option<MaybeRelocatable>>,
 }
 
 impl BusinessLogicSyscallHandler {
     pub fn new() -> Result<Self, SyscallHandlerError> {
         let events = Rc::new(RefCell::new(Vec::new()));
         let tx_execution_context = Rc::new(RefCell::new(TransactionExecutionContext::new()));
+        let general_config = RefCell::new(StarknetGeneralConfig::new());
+        let tx_info_ptr = RefCell::new(None);
```

Comment:
does and general_config field change during execution? because if not, then it is not necessary to set it as RefCell